lib/js/jsconsole.nim

```diff
@@ -107,7 +107,7 @@ since (1, 5):
     const
       loc = instantiationInfo(fullPaths = compileOption("excessiveStackTrace"))
       msg = getMsg(loc, astToStr(assertion)).cstring
-    {.line: loc.}:
+    {.line: instantiationInfo(fullPaths = true).}:
```

Comment:
What is the purpose of this change?

Reply:
the line pragma always expected absolute path. even it can handle relative path in down stream, it will occurs several file io error on compile time.

Reply:
Got it, thanks. Since you're changing this line anyway, please use `{.line.}` (without any parameter) instead -- it does the same thing, while being much more efficient.